Modules/CodeTrek/Resources/views/index.blade.php

```diff
@@ -135,7 +135,12 @@ class="btn-sm btn-primary mr-1 text-decoration-none"
                                 </td>
                                 <td>
                                     @php
-                                        $daysInCodetrek = now()->diffInDays($applicant->start_date);
+                                        $intern_date = Carbon\Carbon::parse($applicant->internship_start_date);
```

Comment:
In Portal we use camelCase for variable naming in PHP. So please update the name like suggested below
```suggestion
                                        $internshipStartDate = Carbon\Carbon::parse($applicant->internship_start_date);
```

Reply:
okay sir